sda/internal/config/config.go

```diff
@@ -51,9 +52,22 @@ type Config struct {
 	Auth         AuthConf
 }
 
+type Grpc struct {
+	CACert      string
+	ClientCert  string
+	ClientKey   string
+	ClientCreds credentials.TransportCredentials
+	ServerCert  string
+	ServerKey   string
```

Comment:
Technically this should look like this:

```
	CACert      string
	ClientCreds credentials.TransportCredentials
	TLSCert     string
	TLSKey      string
```

But that would require changes to the server setup function as well so we leave it for now.